Formula/nginx-unit.rb

```diff
@@ -0,0 +1,223 @@
+class NginxUnit < Formula
+  desc "Dynamic web and application server for multiple languages"
+  homepage "https://unit.nginx.org/"
+  url "https://unit.nginx.org/download/unit-1.1.tar.gz"
+  sha256 "7c66365f5ea87e7e8903bac9d639676947fe2ab7aa799ce35defc426b3409ee0"
+
+  depends_on "openssl"
+  depends_on "pcre"
+
+  def install
+    openssl = Formula["openssl"]
+    pcre = Formula["pcre"]
+
+    # The modules directory is where unitd loads modules. Compile unitd with
+    # prefix-relative directory for loading and install moduless to keg-local
+    # to be linked in so other formulae can also link modules.
+    system "./configure", "--prefix=#{prefix}",
+                          "--modules=#{HOMEBREW_PREFIX}/lib/unit",
+                          "--state=#{var}/unit",
+                          "--log=#{var}/log/unit.log",
+                          "--pid=#{var}/run/unit.pid",
+                          "--control=unix:#{var}/run/unit.control.sock",
+                          "--cc-opt=-I#{pcre.opt_include} -I#{openssl.opt_include}",
+                          "--ld-opt=-L#{pcre.opt_lib} -L#{openssl.opt_lib}"
+
+    system "make", "install"
+
+    libunit = lib/"unit"
+    libunit.mkpath
```

Comment:
this line isn't needed since you use `.install` below which does a `mkdir -p` first

Reply:
Neat!